convert.py

```diff
@@ -104,6 +115,28 @@ def GenerateAudioContentForText(text):
             break
     
 
+def eupub_to_chapters():

```

Comment:
Some function documentation above this would be great :)

Reply:
ack